lib/index.js

```diff
@@ -86,22 +87,27 @@ Optimizely.prototype.initialize = function() {
  */
 
 Optimizely.prototype.track = function(track) {
+  var opts = this.options;
+  var eventProperties = track.properties();
   // Use the new-style API (which is compatible with Classic and X)
   var payload = {
     type: 'event',
     eventName: track.event(),
-    tags: track.properties()
+    tags: eventProperties
   };
 
-  if (payload.tags.revenue) {
+  if (eventProperties.revenue && opts.sendRevenueOnlyForOrderCompleted) {
+    if (track.event() === 'Order Completed') {
+      payload.tags.revenue = Math.round(payload.tags.revenue * 100);
```

Comment:
Just to add clearity on the bug / issue 
https://github.com/segment-integrations/analytics.js-integration-optimizely/blob/master/lib/index.js#L93
https://github.com/segment-integrations/analytics.js-integration-optimizely/blob/master/lib/index.js#L109

track.properties is sent off in two locations to the snippet and full stack SDK. Both need the  formatted revenue fix.